### DISCUSSION TITLE
FPS showing differently when dragging to second monitor

### DISCUSSION THREAD

**Discussion by bozzis** [ppy/osu#28330] [Q&A]
### Type

Game behaviour

### Bug description

I have two exact same monitors, both running at 144Hz with displayport connection.
When I drag windowed osu from one to the other, the fps display change from 144 to 1000.
Gameplay is the same for both though, I don't feel differences at all, so maybe a visual glitch?
Doesn't happen for OpenGL, only for Direct3D.
On the video, I am dragging osu window from one monitor to the other, check the fps at the bottom right.
Maybe it's related to one display being set as main in Windows, I don't know.

### Screenshots or videos

https://github.com/ppy/osu/assets/24719518/13ee558e-82f0-4465-bab6-78e467e61cee

### Version

2024.521.2

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15447105/compressed-logs.zip)

**Comment by shinmorisawa** [comment]
this is probably a direct3d problem, not an osu problem
